Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
+
+## Making a contribution
+1. Fork the repo which you want to work on (eg. https://github.com/take2rohit/sih_number_plate is original repo where you want to contribute) 
+1. Clone the forked repository (not the above repo link)
+`git clone https://github.com/your_userid/sih_number_plate`
+3. Do your changes whatever you wish and commit 
```

Comment:
Do the changes and commit. Let's try to be grammatically correct from the beginning.